operators/src/adapters/raster_subquery/raster_subquery_adapter.rs

```diff
@@ -543,8 +553,12 @@ pub fn identity_accu<T: Pixel>(
 ) -> impl Future<Output = Result<RasterTileAccu2D<T>>> {
     crate::util::spawn_blocking(move || {
         let output_raster = EmptyGrid2D::new(tile_info.tile_size_in_pixels).into();
-        let output_tile =
-            RasterTile2D::new_with_tile_info(query_rect.time_interval, tile_info, output_raster);
+        let output_tile = RasterTile2D::new_with_tile_info(
+            query_rect.time_interval,
+            tile_info,
+            output_raster,
+            CacheHint::unlimited(),
```

Comment:
`CacheHint::unlimited()` is the initial state in every accumulator. It is then merged with the cache hint of all the tiles that go into the accumulator. If all are also unlimited cacheable, it stays unlimited. If any is non cacheable or do expire, the final cache hint will reflect this accordingly.

The only requirement here is that at least one tile is fed into the accumulator. This should always be the case for non-empty query rectangles, because at the very least we get empty tiles.